docs/modules/ROOT/pages/documentation/ecosystem/intellij-plugin/payara-micro.adoc

```diff
@@ -0,0 +1,108 @@
+
+* Enter the groupId, artifactId, etc to set up the project, and then click *Next*:
+
+image::intellij-plugin/payara-micro/project-gav-details.png[Enter GAV]
+
+* On the Project name and Location panel, enter *sample-micro-app* as the project name, 
+set the Project destination folder and click *Finish*:
+
+image::intellij-plugin/payara-micro/project-name-details.png[Enter Project name]
+
+* When you click Finish, the IDE creates the following projects from the Payara Micro maven archetype:
+
+image::intellij-plugin/payara-micro/generating-micro-project.png[Generating Micro project]
+
+* Once the project is generated, you may either add the project to the current workspace or 
+open in a new window:
+
+image::intellij-plugin/payara-micro/generated-micro-project.png[Generated Micro project]
+
+* As you can see, the project has a basic structure with *src\main\java* and *src\main\webapp* folders:
+
+image::intellij-plugin/payara-micro/project-structure.png[Project source tree]
+
+Payara Micro application can be managed via IntelliJ Run/Debug configuration or Actions, 
+where Run configuration can be customized as per developer choice and Actions are quick & easy to use:
+
+[[run-micro-project-via-config]]
+== Run the Payara Micro Application via configuration:
+
+The IntelliJ IDEA needs a run configuration to build the artifacts and run the Payara Micro application.
+
+* From the main menu, select *Run | Edit Configurations*:
+
+image::intellij-plugin/payara-micro/create-run-config.png[Create Config]
+
+* In the *Run/Debug Configurations* dialog, click *+*, expand the *Payara Micro* node, and select *Maven*:
+
+image::intellij-plugin/payara-micro/create-maven-run-config.png[Create Micro Maven Config]
+
+* Now add the Before launch tasks and assign a new artifact, click + on the left-hand pane, 
+choose the *Build Artifacts* type from the drop-down list:
+
+image::intellij-plugin/payara-micro/build-artifact.png[Build artifact]
+
+* In the *Select Artifacts* dialog, select to build the *sample-micro-app:war* artifact:
+
+image::intellij-plugin/payara-micro/choose-artifact.png[Choose artifact]
+
+* Now save the configuration:
+
+image::intellij-plugin/payara-micro/save-config.png[Save Configuration]
+
+* To start the Payara Micro instance, click the run button in the top-right panel:
+
+image::intellij-plugin/payara-micro/run-config.png[Run Configuration]
+
+* After running the Micro application, you may see the log in the OUTPUT window:
+
+image::intellij-plugin/payara-micro/run-log.png[Payara Micro log]
+
+
+
+=== Debugging the Payara Micro Application
+
+* Instead of pressing the *Run* button, you can instead click *Debug* from similar 
+locations to automatically start the micro instance and attach the debugger to the application:
+
+image::intellij-plugin/payara-micro/debug-config.png[Debug Configuration]
+
+* After debugging the Payara Micro application, you may see the log in the *Debug ? Console* window:
```

Comment:
```suggestion
* After debugging the Payara Micro application, you may see the log in the *Debug > Console* window:
```

Reply:
Seems something went wrong from copying one editor to other as I was using this symbol →